src/components/file-picker/file-picker.jsx

```diff
@@ -0,0 +1,162 @@
+import PropTypes from 'prop-types';
+import React, { useState, useRef } from 'react';
+import Icon from '../icon/icon';
+import useError from '../../hooks/useError';
+import styles from './file-picker.css';
+import iconUpload from '../../svgs/icon-cloud-upload-negative.svg';
+import iconFileDefault from '../../svgs/icon-file-default.svg';
+
+const FilePicker = (props) => {
+  const {
+    dropzoneClasses,
+    labelClasses,
+    fileClasses,
+    fileListClasses,
+    id,
+    title,
+    receiveFilesChanged,
+    validator,
+    ...rest
+  } = props;
+
+  const [files, setFiles] = useState([]);
+  const [getError, setError, validate] = useError(validator);
+  const [highlight, setHighlight] = useState(false);
+  const inputFile = useRef(null);
+
+  const setFilesChanged = (selectedFiles) => {
+    // Accounts for case where User's clicked 'Cancel' on native file dialog
+    if (selectedFiles.length) {
+      const currentFiles = [];
+      Array.from(selectedFiles).map(file => currentFiles.push(file));
+      const errMsg = validate(currentFiles);
+      if (errMsg.length) {
+        setError(errMsg);
+      } else {
+        setError('');
+        setFiles(currentFiles);
+        if (props.receiveFilesChanged) {
+          props.receiveFilesChanged.call(this, currentFiles);
+        }
+      }
+    }
+  };
+
+  const onFilesChanged = (e) => {
+    const selectedFiles = e.currentTarget.files;
+    setFilesChanged(selectedFiles);
+  };
+
+  const onRemoveFile = (e, file) => {
+    e.preventDefault();
+    const currentFiles = files.filter(f => f.name !== file.name);
+    setFiles(currentFiles);
+    if (props.receiveFilesChanged) {
+      props.receiveFilesChanged.call(this, currentFiles);
+    }
+  };
+
+  const onLabelKeypress = (e) => {
+    e.preventDefault();
+    if (e.key === 'Enter') {
+      inputFile.current.click();
+    }
+  };
+
+  // This prevents the dropEffect going from "copy" to "none"
+  // (in chrome, cursor turns from a green plus to an arrow)
+  // https://github.com/react-dropzone/react-dropzone/blob/master/src/index.js#L533
+  const fixDropEffect = (e) => {
+    e.stopPropagation();
+    if (e.dataTransfer) {
+      try {
+        e.dataTransfer.dropEffect = 'copy';
+      } catch {} /* eslint-disable-line no-empty */
+    }
+  };
+
+  const onDragOver = (e) => {
+    e.preventDefault();
+    fixDropEffect(e);
+  };
+
+  const onDragLeave = () => {
+    setHighlight(false);
+  };
+
+  const onDrop = (e) => {
+    e.preventDefault();
+    e.stopPropagation();
+    const droppedFiles = e.dataTransfer.files;
+    setFilesChanged(droppedFiles);
+    setHighlight(false);
+  };
+
+  const getFilesList = () => {
+    if (files.length) {
+      return files.map((file) => {
+        return (
+          <section className={styles['file-list-button']} key={file.name}>
```

Comment:
Let's convert this list of `<section>`s into a list of `<li>`s

Reply:
Fixed in: 07b8b2f7e9d87470b39abc14da85488374941841